code/src/gfx_options.c

```diff
@@ -66,6 +67,23 @@ void InitOptions(void) {
     strcpy(options[opNum].alternatives[altNum++], "Skip (Keep SFX)");
     strcpy(options[opNum].description, "Toggle skipping the automatic replay after\nyou play a song.");
     options[opNum++].optionPointer = &gExtSaveData.option_SkipSongReplays;
+
+    // Free Camera Control
+    altNum = 0;
+    strcpy(options[opNum].name, "Free Camera Control");
+    if (gSettingsContext.freeCamera) {
+        strcpy(options[opNum].alternatives[altNum++], "Normal");
+        strcpy(options[opNum].alternatives[altNum++], "Invert Y-Axis");
+        strcpy(options[opNum].alternatives[altNum++], "Invert X-Axis");
+        strcpy(options[opNum].alternatives[altNum++], "Invert Both");
+        strcpy(options[opNum].description,
+               "Change the input directions of the free camera.\n\nThis feature is not available on old 3DS systems.");
+    } else {
+        strcpy(options[opNum].alternatives[altNum++], "Disabled");
+        strcpy(options[opNum].description, "Free camera is disabled, enable it in\n\"Misc Settings\".\nThis feature is "
+                                           "not available on old 3DS systems.");
+    }
+    options[opNum++].optionPointer = &gExtSaveData.option_FreeCamControl;
```

Comment:
You can wrap the entire option in the if statement so it doesn't show up when Free Cam is off. Empty/unused options are already checked for, and ignored.